### DISCUSSION TITLE
Scythe Anticheat v2.24.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#152] [Announcements]
Scythe Anticheat is now at version v2.24.0!

Supported Minecraft Versions: 1.21.20

# Changelog
- Added wind charge to ReachA excluded entities list
- Deprecated FlyA module as Mojang broke the method used
- Changed KillauraC minimum entity count to 3 to avoid false positives when the server is lagging
- Players can now have a default chat rank. You can use this feature by modifying the `defaultTag` property in `config.js`
- Fixed false positives with KillauraB
- `!vanish` puts you back in your previous gamemode when you exit out of it
- Fixed village raids not working
- Bug fixes
- Performance optimizations

By downloading this addon, you agree to our [license](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/blob/main/LICENSE)
Questions? Join our [discord](https://discord.gg/pKyu95mYCT)!

We are currently requesting feedback about Scythe, if you have around 3 to 5 minutes, please consider filling out the [Scythe Anticheat Feedback Form!](http://docs.google.com/forms/d/1YFj_hAj1U2uJWtSqaUE69djVZsQJhprMmyx9ayKTxIg)

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-Anticheat/releases/tag/v2.24.0'>Scythe Anticheat v2.24.0</a>.</em>

**Comment by ExtraBytez** [comment]
default chat rank is a game changer thanks @MrDiamond64
